src/test/java/org/jenkinsci/plugins/github/migration/MigratorTest.java

```diff
@@ -82,7 +82,7 @@ public void shouldMigrateCredentials() throws Exception {
     @Test
     @LocalData
     public void shouldLoadDataAfterStart() throws Exception {
-        assertThat("should load 3 configs", GitHubPlugin.configuration().getConfigs(), hasSize(2));
+        assertThat("should load 2 configs", GitHubPlugin.configuration().getConfigs(), hasSize(2));
```

Comment:
the config has 2 configs as validated by the assert :)

Reply:
I am correct assuming this configuration is test data that will not change? I mean, there is no extension point that can contribute more configs than the expected number based on the plugins in the classpath?

Reply:
the config is test data so should not change.

https://github.com/jenkinsci/github-plugin/blob/dcba2669fa8830dbc23efadeb947d02f271231e1/src/test/resources/org/jenkinsci/plugins/github/migration/MigratorTest/shouldLoadDataAfterStart/github-plugin-configuration.xml#L1-L16



if it does change due to any dependencies or re-work then it should be obvious at the point in time that is done in this plugin.